.github/workflows/pull_request.yml

```diff
@@ -31,13 +31,22 @@ defaults:
     shell: bash
 
 jobs:
+  os-prep:
+    runs-on: ubuntu-latest
+    outputs:
+      os: ${{ steps.set-os.outputs.operating-systems }}
+    steps:
+      - name: OS Preparation
+        id: set-os
+        uses: lampajr/kogito-pipelines/.ci/actions/os-preparation@BXMSPROD-1770_os_preparation
```

Comment:
here don't forget to use kiegroup instead of your github

Reply:
yeah sure, once the https://github.com/kiegroup/kogito-pipelines/pull/590 will be merged in `kogito-pipelines` I'll fix here checking everything will be fine with the correct configuration.